manifest.yaml

```diff
@@ -97,6 +97,12 @@ postprocess:
      EOF
      fi
 
+     # TEMPORARY: Fix file permission for cpictl until fix is backported to RHEL 8.6
+     # See https://bugzilla.redhat.com/show_bug.cgi?id=2024102
+     if [ "$(uname -m)" == "s390x" ]; then
+     chmod 755 /usr/lib/s390-tools/cpictl
+     fi
+
```

Comment:
I'd suggest enhancing this to check if the permissions are set correctly before applying this workaround. Maybe emit a message too, so we can know when we can remove the workaround.

Reply:
I added a check for this. I would also like to reserve the right to miss that message in between all the output of a `cosa build` unless i specifically search for it.